mk/header.mk

```diff
@@ -1,7 +1,7 @@
 # Clear vars used by this make system
 define HEADER
 CLEAN :=
-TARGETS = $(TRGs) $(DTAs) $(SQLs) $(BNDDs) $(PFs) $(LFs) $(DSPFs) $(PRTFs) $(CMDs) $(SQLs) $(MODULEs) $(SRVPGMs) $(PGMs) $(MENUs) $(PNLGRPs) $(QMQRYs) $(WSCSTs) $(MSGs)
+TARGETS = $(TRGs) $(DTAs) $(SQLs) $(BNDDs) $(PFs) $(LFs) $(DSPFs) $(PRTFs) $(CMDs) $(MODULEs) $(SRVPGMs) $(PGMs) $(MENUs) $(PNLGRPs) $(QMQRYs) $(WSCSTs) $(MSGs)
```

Comment:
Just curious why you are dropping SQL from TARGETs?

Reply:
`TARGETS = $(TRGs) $(DTAs) **$(SQLs)**`
Because there is a duplicate one at the begining